packages/storefront-lwc/src/modules/commerce/coupons/gqlMutations.js

```diff
@@ -0,0 +1,75 @@
+import gql from 'graphql-tag';
+
+export const ADDCOUPONMUTATION = gql`
+    mutation submitCoupon($couponCode: String!) {
+        submitCoupon(couponCode: $couponCode) {
+            basketId
+            customerId
+            getBasketMessage
+            totalProductsQuantity
+            shipmentId
+            shipmentTotal
+            selectedShippingMethodId
+            products {
+                productId
+                itemId
+                quantity
+                productName
+                price
+                image
+                }
+            orderTotal
+            orderLevelPriceAdjustment {
+                itemText
+                price
+            }
+            shippingTotal
+            shippingTotalTax
+            taxation
+            taxTotal
+            couponItems {
+                valid
+                code
+                couponItemId
+                statusCode
+            }
+        }
+    }
+`;
+
+export const REMOVECOUPONMUTATION = gql`
+    mutation removeCoupon($couponItemId: String!) {
+        removeCoupon(couponItemId: $couponItemId) {
+            basketId
```

Comment:
REMOVECOUPONMUTATION looks very similar to ADDCOUPONMUTATION. Is there anyway to abstract them out and make it reusable?

Reply:
Good point, fixing it.